src/biblebot/triggers.py

```
@@ -0,0 +1,78 @@
+        translation = trans_group.lower() if trans_group else default_translation
+        return passage, translation
+    return None
+
+
+def _try_direct_match(body: str, default_translation: str) -> tuple[str, str] | None:
+    return _match_reference(body, REFERENCE_PATTERNS, "fullmatch", default_translation)
+
+
+def detect_trigger(
+    body: str,
+    formatted_body: str | None,
```

Ruff (0.15.10)

[warning] Unused function argument: `formatted_body`

(ARG001)